### PR TITLE
Reduce memory footprint by not caching primitive attributes

### PR DESCRIPTION
**Pull request by hansonchar**

### CODE REVIEW

File: code-generator/src/main/resources/templates/model-pojo.ftl [hunk 1]
```diff
@@ -48,6 +50,15 @@ public class ${shape.name} extends com.amazonaws.sdk.AwsPojo<${shape.name}> {
      * @return the current value of this property
      */
     public ${member.shape.javaType} get${member.name}() {
+      <#if member.shape.isPrimitiveType()>
+        Value value = get("${member.name}");
+        if (value == null || value.isNull()) {
+            return null;
+        } else {
+            ${member.shape.javaType} out = Value.unwrap(value);
```
Comment (david-at-aws) [inline code-generator/src/main/resources/templates/model-pojo.ftl]: return Value.unwrap(value) to save on some code?

Comment (hansonchar) [inline code-generator/src/main/resources/templates/model-pojo.ftl]: Unfortunately that won't compile.

Comment (david-at-aws) [inline code-generator/src/main/resources/templates/model-pojo.ftl]: Ah, that makes sense. It's either this or explicitly calling Value.<${member.shape.javaType}>unwrap(value), and I prefer this. Looks good!

Comment (hansonchar) [inline code-generator/src/main/resources/templates/model-pojo.ftl]: Actually it works with

>return Value.unwrap(); 

It only didn't work when I tried the 1-liner:

>return value == null || value.isNull() ? null : Value.wrap(value);

Anyhow, the latest pull request should have the change included.

**Pull request merged**